src/sql/workbench/browser/designer/designerPropertiesPane.ts

```diff
@@ -42,6 +49,23 @@ export class DesignerPropertiesPane {
 		return this._objectPath;
 	}
 
+	private set showDescription(value: boolean) {
```

Comment:
Let's make it work like this: The description component is always visible, before any field is selected, leave both title and description blank, otherwise show whatever is available.

Reply:
In that case, I think that's very bad UX because there can be a case when I select a property with a description and then subsequently select another property without one, the UI shows the focused element is the new property but we still see the old description.

Reply:
in that case the description will be empty right? the title will be the newly focused property name, user can easily tell the description area has been refreshed.